<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{rgb(170, 44, 33);}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>347</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight SemiConde</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Practical Work â„–9</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pbCreate">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight Condensed</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Create Object</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbShow">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight Condensed</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Show Object</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbExit">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight Condensed</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Exit the program</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formWidget" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>371</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing">
      <number>20</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="idLb">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Enter ID: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="idLn"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="surnameLb">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Enter Surname:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="surnameLn"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="firstNameLb">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Enter First Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="firstNameLn"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lastNameILb">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Enter Last Name: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lastNameLn"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="addressLb">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Enter Address: </string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="addressLn"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="phoneNumberLb">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Enter Phone Number: </string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="phoneNumLn"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="medNumLb">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Enter Medical Number: </string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="medNumLn"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="diagnosisLb">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Enter Diagnosis: </string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="diagnosisLn"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight Condensed</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Input all the data:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formWidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_6">
     <property name="verticalSpacing">
      <number>20</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>440</y>
      <width>168</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="pbConfirm">
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight SemiConde</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Confirm and create</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/1445149.png</normaloff>:/rec/img/1445149.png</iconset>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/72647.png</normaloff>:/rec/img/72647.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/rec/img/123.png</normaloff>:/rec/img/123.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbCreate</sender>
   <signal>clicked()</signal>
   <receiver>formWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
